### PR TITLE
IGAPP-1327: Consistent translations

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by steffenkleinle** [comment]
This PR
* Removes tunews keys and instead uses common news ones
* Changes German wording from formal to informal (sie -> du) and removes affected translations
* Moves the key "textCopied" to common and renames to "copied"

**Review by LeandraH — COMMENTED** [review]
Nice! Tested on emulated iOS

**Review by LeandraH — APPROVED** [review]
Arg

File: web/src/components/CityContentToolbar.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const CityContentToolbar = (props: CityContentToolbarProps) => {
   return (
     <Toolbar iconDirection={iconDirection} hasDivider={hasDivider}>
       {children}
-      <Tooltip text={t('cityNotCooperating:textCopied')} flow='up' active={linkCopied} trigger='click'>
+      <Tooltip text={t('common:textCopied')} flow='up' active={linkCopied} trigger='click'>
```
Comment (sarahsporck) [inline web/src/components/CityContentToolbar.tsx:35]:
```suggestion
      <Tooltip text={t('common:copied')} flow='up' active={linkCopied} trigger='click'>
```

**Review by sarahsporck — CHANGES_REQUESTED** [review]
Tested on android emulator and web

File: web/src/components/CityContentToolbar.tsx (hunk 1 above)
Comment (steffenkleinle) [reply web/src/components/CityContentToolbar.tsx:35]: Good catch, thanks!

**Review by sarahsporck — APPROVED** [review]

**Pull request merged**